rules/macos/execution_curl_spawned_from_installer_package.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/02/23"
+maturity = "production"
+updated_date = "2021/02/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies when the built in macOS Installer program is used to install a package that spawns a shell and runs
+the cURL command.
+"""
+false_positives = [
+    "Custom organization-specific macOS packages that use .pkg files to run cURL could trigger this rule.",
```

Comment:
```suggestion
    "Custom organization-specific macOS packages that use .pkg files to run cURL could trigger this rule. If known behavior is causing false positives, it can be excluded from the rule.",
```

Reply:
I think we can take the `note` section out and break it up into the false positives and description sections.